core_screenshot_config.yaml

```diff
@@ -184,6 +184,11 @@ quaternary:
     overseerScript: page.manualWait(); await page.waitForDelay(60000); page.done();
     message: wait for IA quaternary
 
+  PR:
+    overseerScript: await page.waitForNavigation({waitUntil:"domcontentloaded"}); await page.waitForSelector("#ember906");page.click("#ember906");page.done();
```

Comment:
```suggestion
    overseerScript: >
      await page.waitForNavigation({waitUntil:"domcontentloaded"});
      await page.waitForSelector("#ember906");
      page.click("#ember906");
      page.done();
```

Reply:
thank you! Just nit, for readability I've been trying to keep these config files to an easily viewable width so just adding some newlines here

Reply:
of course! fixed